packages/fx-core/resource/yaml-schema/yaml.schema.json

```diff
@@ -1782,6 +1782,10 @@
             "configurationId": {
               "description": "Required. The configuration id of created OAuth registration.",
               "$ref": "#/definitions/envVarName"
+            },
+            "applicationIdUri": {
+              "description": "Required. The Application ID URI of created OAuth registration.",
+              "$ref": "#/definitions/envVarName"
```

Comment:
This is not required, and only for SSO, if it is the normal OAuth, there is no identify uri